modules/src/food.py

```diff
@@ -0,0 +1,93 @@
+import os
+import json
+import requests
+import random
+import difflib
+
+import config
+from templates.generic import *
```

Comment:
Let's import the module that we need explicitly rather than importing everything, very dangerous :)